hooks/useConnectWallet.tsx

```diff
@@ -1,21 +1,15 @@
-import trackLoginPoints from "@/lib/stack/trackLoginPoints";
-import { useEffect } from "react";
-import { useAccount, useConnect } from "wagmi";
+import { usePrivy } from '@privy-io/react-auth';
+import { Address } from "viem";
 
-const useConnectWallet = () => {
-  const { connectors, connectAsync } = useConnect();
-  const connector = connectors[0];
-  const { address } = useAccount();
+interface ConnectWalletReturn {
+  address: Address | undefined;
+}
 
-  useEffect(() => {
-    if (!address) return;
-    trackLoginPoints(address);
-  }, [address]);
-
-  const connectWallet = () => connectAsync({ connector });
+const useConnectWallet = (): ConnectWalletReturn => {
```

Comment:
Is this hookName accurately representing it's functionality? 🤔 
- actual: `useConnectWallet` - I can `connect a wallet`
- expected: `useConnectedWallet` - I can `access the connected wallet`
```suggestion
const useConnectedWallet = (): ConnectWalletReturn => {
```